[V2] Address linter errors in az-analyze tool

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:

`golangci-lint` is failing with the following errors for the `az-analyze` tool

```bash
$ golangci-lint run -E=gofmt,deadcode,unused,varcheck,ineffassign,revive,misspell,exportloopref,asciicheck,bodyclose,depguard,dogsled,durationcheck,errname,forbidigo -D=staticcheck --timeout=30m0s
WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive.
cmd/az-analyze/cmd/azv.go:47:4: use of `fmt.Println` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)
                        fmt.Println("No azVolume was found")
                        ^
cmd/az-analyze/cmd/azva.go:52:4: use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)
                        fmt.Printf("only one of the flags is allowed.\n" + "Run 'az-analyze --help' for usage.\n")
                        ^
cmd/az-analyze/cmd/azva.go:60:6: use of `fmt.Println` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)
                                        fmt.Println("No azVolumeAttachment was found")
                                        ^
cmd/az-analyze/cmd/azva.go:67:6: use of `fmt.Println` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)
                                        fmt.Println("No azVolumeAttachment was found")
                                        ^
cmd/az-analyze/cmd/get.go:33:4: use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)
                        fmt.Printf("You must specify the type of resource to \"%s\"\n", cmd.Name())
                        ^
cmd/az-analyze/cmd/get.go:34:4: use of `fmt.Printf` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)
                        fmt.Printf("Run 'az-analyze --help' for usage.\n")
                        ^
cmd/az-analyze/cmd/azv.go:39:3: S1021: should merge variable declaration with assignment on next line (gosimple)
                var result []AzvResource
                ^```

**What you expected to happen**:

No linter errors.

**How to reproduce it**:

**Anything else we need to know?**:

**Environment**:
- CSI Driver version:
- Kubernetes version (use `kubectl version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -7,3 +7,9 @@ linters:
   - structcheck
   enable:
   - golint
+
+issues:
+  exclude-rules:
+    - path: 'cmd/az-analyze'
+      linters:
+        - forbidigo

--- a/cmd/az-analyze/cmd/azv.go
+++ b/cmd/az-analyze/cmd/azv.go
@@ -19,10 +19,11 @@ package cmd
 import (
 	"context"
 	"fmt"
+	"os"
+
 	"github.com/olekukonko/tablewriter"
 	"github.com/spf13/cobra"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"os"
 	v1beta1 "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/azuredisk/v1beta1"
 	consts "sigs.k8s.io/azuredisk-csi-driver/pkg/azureconstants"
 )
@@ -36,8 +37,7 @@ var azvCmd = &cobra.Command{
 		pod, _ := cmd.Flags().GetString("pod")
 		namespace, _ := cmd.Flags().GetString("namespace")
 
-		var result []AzvResource
-		result = GetAzVolumesByPod(pod, namespace)
+		result := GetAzVolumesByPod(pod, namespace)
 
 		// display
 		if len(result) != 0 {

